src/lib/components/hamburger-menu/menu.svelte

```diff
@@ -48,7 +48,7 @@
         {/if}
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-9 px-3">
```

Comment:
I feel this margin plus a padding is a bit too large what do you think?

Reply:
Maybe I can reduce the padding. Just wanted to ask you regarding this, since there only two options in the menu